packages/theme/pages/MyAccount.vue

```diff
@@ -142,6 +142,11 @@ export default {
   }
   &__logout {
     cursor: pointer;
+    outline: none;
```

Comment:
Disabling the button's outline is an accessibility antipattern (see: https://www.a11yproject.com/posts/never-remove-css-outlines/)

Reply:
Good point!